.github/workflows/run_tests.yaml

```diff
@@ -1,25 +1,33 @@
 ## CI - Player And Plugin Tests
```

Comment:
this file was commented on purpose. the tests are in a separate PR which is not merged yet.

Reply:
I understand, but you can’t leave it for the time being without tests, anyway you can change it later